src/client/kotlin/tech/thatgravyboat/skyblockapi/api/events/render/RenderWorldEvent.kt

```diff
@@ -0,0 +1,35 @@
+package tech.thatgravyboat.skyblockapi.api.events.render
+
+import com.mojang.blaze3d.vertex.PoseStack
+import net.fabricmc.fabric.api.client.rendering.v1.WorldRenderContext
+import net.minecraft.client.Camera
+import net.minecraft.client.renderer.MultiBufferSource
+import tech.thatgravyboat.skyblockapi.api.events.base.SkyBlockEvent
+import tech.thatgravyboat.skyblockapi.utils.extentions.pushPop
+
+open class RenderWorldEvent(
+    val poseStack: PoseStack,
```

Comment:
we should just have the context and then have getters for the posestacks, camera and eta.